src/main/java/com/pmm/budgetcalculator/expensing/service/ExpenseTypeService.java

```diff
@@ -19,22 +21,35 @@ public List<ExpenseType> getAllExpenseTypes() {
     }
 
     public ExpenseType getExpenseTypeById(Long id) {
-        return expenseTypeRepository
-                .findById(id)
-                .orElseThrow(() -> new RuntimeException("Expense not found for Id:" + id));
+        return expenseTypeRepository.findById(id).orElseThrow(() -> new RuntimeException("not found"));
     }
+
     public ExpenseType createExpenseType(ExpenseType expenseType) {
         return expenseTypeRepository.save(expenseType);
     }
-    public void removeExpenseTypeById(Long id){
+
+    public void removeExpenseTypeById(Long id) {
         if (expenseTypeRepository.existsById(id)) {
             expenseTypeRepository.deleteById(id);
-        }
+        } else ResponseEntity.notFound().build();
+    }
 
-//    public List<ExpenseType> findByCriteria(String expenseType) {
-//        if (Objects.nonNull(expenseType)) {
-//            return expenseTypeRepository.findByCriteria(expenseType);
-//        }
-//        return expenseTypeRepository.findAll();
+    public ResponseEntity<ExpenseType> updateExpenseType(Long id, ExpenseType expenseTypeDetails) {
```

Comment:
Do it like every other method. Return ExpenseType and in controller wrap in ResponseEntity<>